docs/pages/enroll-resources/application-access/jwt/grafana.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Using JWT authentication with Grafana
+description: How to use JWT authentication with Grafana
+tags:
+ - how-to
+ - zero-trust
+ - infrastructure-identity
+---
+
+{/* lint disable page-structure remark-lint */}
+
+This guide will help you configure Grafana [JWT
+authentication](https://grafana.com/docs/grafana/latest/setup-grafana/configure-access/configure-authentication/jwt/)
+with Teleport.
+
```

Comment:
While this guide is still new, I think it makes sense to add a quick `## How it works` section and un-disable the structure linter. This section goes a long way towards helping users understand a how-to guide, since it explains what exactly an admin is setting up.